readme.txt

```diff
@@ -85,6 +85,13 @@ Release and roadmap notes available on the [WooCommerce Developers Blog](https:/
 
 == Changelog ==
 
+== 5.9.0 - 2021-09-13 =
+
+#### Enhancements
+
+- Add extensibility point for extensions to filter payment methods. ([4668](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4668))
```

Comment:
Not blocking, but I expected #4740 to be listed as well given that it fixes a visible bug.